README.md

```diff
@@ -39,6 +39,19 @@ Prior to using stackhpc-appliances, ensure the virtual environment is activated:
 
     source venv/bin/activate
 
+### ALaSKA Environments
+
+There are two ALaSKA environments - Production and Alt-1. This repository
+supports both of these environments. To use a specific environment, ensure that
+any OpenStack authentication environment variables reference the correct
+environment. When executing playbooks, be sure to use the correct Ansible
+inventory:
+
+* `ansible/inventory` is for production
```

Comment:
Can we call this inventory-prod/uction please?

Reply:
I wanted to remain backwards compatible. Plus, it follows the pattern of alaska/alaska-alt-1